### ISSUE TITLE
Unable to install projects

### ISSUE THREAD

**Issue by SirenGlitch** [apk-editor/APK-Explorer-Editor#25]
I am making a "simple" sound modification to the game Dino T-rex and whenever I try and install the project APK, it fails. Error says invalid APK files selected

**Comment by apk-editor** [comment]
@SirenGlitch 
This issue will be most probably fixed as of v0.25. Can you have a look. Extremely sorry for being too slow in responding to this issue.
PS; Closing this issue. Feel free to reopen it, if the issue still exist.

**Comment by dragos240** [comment]
This still happens with some APKs. The first I tried was EVGo 6.0.67, which worked just fine. All I did was change the version to disable an update nag. The second was UPS 9.7.9.6, which failed to build even with no modifications. I noticed that the new APK was about 10 MB smaller than the original, so perhaps that's part of it.

Steps to reproduce issue:
- Load UPS into the app
- When the list of files pops up, hit build without changing anything
- The build will seem to complete and says it's exported, but it can't be installed, either with this app or the native package installer with "invalid APK files" as the error
